onnxruntime/core/providers/cpu/signal/dft.h

```diff
@@ -1,15 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#ifdef BUILD_MS_EXPERIMENTAL_OPS
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
 
 namespace onnxruntime {
-namespace contrib {
 
 class DFT final : public OpKernel {
   bool is_onesided_ = true;
   int64_t axis_ = 0;
   bool is_inverse_ = false;
+
  public:
   explicit DFT(const OpKernelInfo& info) : OpKernel(info) {
     is_onesided_ = static_cast<bool>(info.GetAttrOrDefault<int64_t>("onesided", 0));
```

Comment:
TODO: change default for "axis" to 1 to match ONNX spec

Reply:
In the past, we would leave the original implementation as a duplicate to make sure the existing customers are still happy in case the standard version introduces incompatibilities. This is the curse of the experimental kernels. I am sure you have considered that.